### PR TITLE
Dashboard: Responsive Navigation + Create Story Button

### PR DESCRIPTION
**Pull request by carloskelly13**
- Adds Create Story button which navigates to Blank Editor View
- Adds Responsive Nav Bar with Dropdown

Kapture 2020-03-20 at 15 02 18

### CODE REVIEW

File: assets/src/dashboard/components/navigation-bar.js [hunk 1]
```diff
@@ -29,8 +29,24 @@ import styled from 'styled-components';
...
+
+const Link = styled.a`
+  font-family: ${({ theme }) => theme.fonts.tab.family};
+  font-size: ${({ theme }) => theme.fonts.tab.size};
+  line-height: ${({ theme }) => theme.fonts.tab.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.tab.letterSpacing};
+  text-decoration: none;
+  margin-left: 40px;
+  color: ${({ theme, active }) =>
+    active ? theme.colors.gray900 : theme.colors.gray500};
```
Comment (mariano-formidable) [inline assets/src/dashboard/components/navigation-bar.js]: Super nit, but I think the figma says the inactive color should be `gray600`:
image

File: assets/src/dashboard/components/navigation-bar.js [hunk 2]
```diff
@@ -29,8 +29,24 @@ import styled from 'styled-components';
  */
 import { useRouteHistory } from '../app/router';
 import { ReactComponent as WebStoriesLogoSVG } from '../images/logo.svg';
-import { BUTTON_TYPES } from '../constants';
+import { BUTTON_TYPES, NEW_STORY_URL } from '../constants';
 import Button from './button';
+import Dropdown from './dropdown';
+
+const Link = styled.a`
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/navigation-bar.js:36]: would this be useful as a component in /typography to be reused?

File: assets/src/dashboard/components/popover-menu/index.js [hunk 3]
```diff
@@ -49,7 +49,6 @@ export const Menu = styled.ul`
   transform: ${({ isOpen }) =>
     isOpen ? 'translate3d(0, 0, 0)' : 'translate3d(0, -1rem, 0)'};
   z-index: ${Z_INDEX.POPOVER_MENU};
-  width: 100%;
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/popover-menu/index.js:52]: thanks for cleaning this up

**Review by BrittanyIRL — APPROVED** [review]
LGTM - just one small question about if the `Link` you wrote for the nav is something that should be reusable.

**Review by mariano-formidable — CHANGES_REQUESTED** [review]
Small bug with the collapsed menu, it looks like the selected page loses it's active color when going from expanded to collapsed nav (and vice versa):

story-color

**Comment by mariano-formidable** [comment]
Just a couple color nits to check out.  Otherwise, looks good to me!

File: assets/src/dashboard/components/navigation-bar.js (hunk 1 above)
Comment (carloskelly13) [reply assets/src/dashboard/components/navigation-bar.js]: Good find!

**Comment by carloskelly13** [comment]
@mariano-formidable Yeah, I think we need to make the text color of the dropdown component darker then.

**Comment by carloskelly13** [comment]
Updated screenshots for new color values.

Screen Shot 2020-03-20 at 4 05 08 PM
Screen Shot 2020-03-20 at 4 05 12 PM

**Review by mariano-formidable — APPROVED** [review]
Excellent!  Looks good to me!

**Review by pbakaus — APPROVED** [review]
Looks good!

**Pull request merged**